src/storage/ResourceStore.ts

```diff
@@ -50,13 +50,13 @@ export interface ResourceStore {
    * @param representation - New representation of the resource.
    * @param conditions - Optional conditions.
    *
-   * @returns A promise resolving when the update is finished.
+   * @returns A promise resolving to a list of resources that were created when the update is finished.
```

Comment:
I am supportive of this idea though, but perhaps rather

```suggestion
   * @returns Identifiers of resources that were possibly modified during the update.
```

Reply:
I have now implemented that idea in https://github.com/solid/community-server/pull/628